exporter/tanzuobservabilityexporter/README.md

```diff
@@ -1,21 +1,29 @@
 # Tanzu Observability (Wavefront) Exporter
 
-This exporter supports sending traces to [Tanzu Observability](https://tanzu.vmware.com/observability).
+This exporter supports sending both traces and metrics to [Tanzu Observability](https://tanzu.vmware.com/observability).
 
 ## Prerequisites
 
 - [Obtain the Tanzu Observability by Wavefront API token.](https://docs.wavefront.com/wavefront_api.html#generating-an-api-token)
 - [Set up and start a Tanzu Observability by Wavefront proxy](https://docs.wavefront.com/proxies_installing.html) and configure it with the API token you obtained.
 - To have the proxy generate [span RED metrics](https://docs.wavefront.com/trace_data_details.html#red-metrics) from trace data, [configure](https://docs.wavefront.com/proxies_configuring.html) the proxy's `customTracingListenerPorts` and use it for the exporter's endpoint.
 
-## Data Conversion
+## Data Conversion for Traces
 
 - Trace IDs and Span IDs are converted to UUIDs. For example, span IDs are left-padded with zeros to fit the correct size.
 - Events are converted to [Span Logs](https://docs.wavefront.com/trace_data_details.html#span-logs).
 - Kind is converted to the `span.kind` tag.
 - Status is converted to `error`, `status.code` and `status.message` tags.
```

Comment:
This line is no longer correct (we missed updating the readme in #7126). It should now read as:

- If a Span's status code is error, a tag of `error=true` is added. If the status also has a description, it's set to `otel.status_description`.

Reply:
Done.